.github/workflows/mason_cli.yaml

```diff
@@ -93,4 +93,4 @@ jobs:
           dart pub global activate pana
 
       - name: Verify Pub Score
-        run: ../../tool/verify_pub_score.sh
+        run: ../../tool/verify_pub_score.sh 100
```

Comment:
👀  on purpose?

Reply:
Yeah until mason is published because pana doesn’t take into account dependency overrides